### ISSUE TITLE
Hard to realign a way during entry, if any mistakes are made

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#1768]
![Screenshot_20220911-171949_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/189549322-3d4d88f9-9d63-4259-8c96-eb64eacf0ec8.jpg)
Here I have purposely started to enter a way, misaligned. It should go directly between the houses, but I skewed it slightly. My point is let's say we put that second lower point on the wrong spot. Well it's going to be very very tough too slide it over a little to the right because, touching it again will just drag up the presets menu, or dragging it right now will drag the road underneath it around too. So I think that is one spot where the app should have a little more flexibility.
I'm talking about the case where we have just put the second node in and we haven't saved any edits or anything.

**Comment by simonpoole** [comment]
Just complete the line and then move it once you are finished, not sure why you believe that should effect the menus, but if it is too near then just pan away a bit.

**Comment by jidanni** [comment]
Okay second tap completes the path. Now we immediately are swung into the menus. That's okay because we then choose path. And then check mark.

Anyway, we are dealing with this area:
```
touch in the same place again - this will finish adding nodes to the way and start the property editor. If the location is within the tolerance zone of a way it will be added as a new node to that way.

touch some other place on the screen - this will create a way segment from the previous location to the current location, the way can be extended by adding further nodes. Touching the last location completes the way and starts the tag editor. If the initial location was near a node that will be used as the first node, and if it was on a way, a new node will be inserted in to that way.

select a menu item see below"
```
Also we are dealing with:
```
Selected objects: Moving a Node or Way

Once you have selected an object, it can be moved. Note that objects can be dragged/moved only when they are selected. Simply drag near (i.e. within the tolerance zone of) the selected object to move it. If you select the large drag area in the preferences, you get a large area around the selected node that makes it easier to position the objec
```
Anyway, the best I could do was:
![Screenshot_20220912-023735_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/189589077-67c2e14d-2db9-444d-905f-38a772face1a.jpg)
And never did figure out a way just to move that node unattached from anything else. Of course I've only tried simple mode so far. Anyway I have a whole new proposal coming up in the next comment..

**Comment by simonpoole** [comment]
What you are doing would seem to be far far too complicated.

Either 

- select the way and drag the whole way (don't drag the geometry improvement handle)

or

- select the end nodes one after the other and drag them.

From a practical pov multi-select as you did it above does the same thing as dragging the way, but it is unnecessary complicated.

**Comment by jidanni** [comment]
Yes, I've got to unglue the final note from the road it is connected to and then I could move it around and without damaging anything. Alas I seem to have forgotten how to exactly do that. But anyway I've got a whole great wonderful new better idea:

Let's think about how a spider makes their web. The spider chooses the place to attach a new node and then attaches it. In no way is the underlying structure distorted during that.

So to solve the current issue, and #1767, a new training wheels mode could be implemented:

While during the addition of way, no distortion of any of the underlying stuff would be permitted.

The way could attach to the underlying stuff but that's all. No relocation of any underlying stuff could occur.

 And any attachment spot will first need to be doubly confirmed, because the user's fat fingers there in the field are very unlikely to get the exact right spot where they should be attached.

Or it should be very easy too grab the point we just put on the map and move it (without moving anything that it is too fastly attached to...)
